datajunction-clients/python/Makefile

```diff
@@ -5,7 +5,7 @@ lint:
 	make check
 
 test:
-	pdm run pytest -n auto --cov=datajunction --cov-report=html -vv tests/ --doctest-modules datajunction --without-integration --without-slow-integration ${PYTEST_ARGS}
```

Comment:
btw, is there a reason the `-cov-report=html` is being removed? (I sometimes glance at it when the build unexpectedly fails in github due to coverage to see what I missed)

Reply:
I mentioned in the summary. That w/o the HTML option I can see the coverage in text mode at the end of the run now.

Reply:
Ohhh, I see, makes sense!